test/opf/opf.jl

```diff
@@ -39,10 +39,6 @@ const PGLIB_CASES = ["14_ieee", "30_ieee", "57_ieee", "89_pegase", "118_ieee"]
         @testset "$(casename)" for casename in PGLIB_CASES
             network = make_basic_network(pglib(casename))
             test_opf_pm(OPF, network)
-
-            data = OPFData(network)
-            test_opfdata(data, network)
-            test_buspair_voltage_bounds(data, network)
```

Comment:
I removed those tests which I believe were redundant with the `OPFData` testset below